src/pybind/mgr/orchestrator_cli/module.py

```diff
@@ -37,7 +37,9 @@ class OrchestratorCli(MgrModule):
         {
             'cmd': "orchestrator service add "
                    "name=svc_type,type=CephString "
-                   "name=svc_arg,type=CephString ",
+                   "name=svc_arg,type=CephString "
+                   "name=pool,type=CephString,req=false "
```

Comment:
Looks like we're trying to compress multiple incompatible commands into a single one.

We're already decided use separate commands for handling osds (see Juan's PR), mons and mgrs, because they needed special command lines. Let's clean up the CLI by creating separate commands:

* `orchestrator nfs add --pool= --namespace=`
* `orchestrator nfs rm ...`
* `orchestrator mds add <name>`
* `orchestrator mds rm ...`
* `orchestrator rgw add <name>`
* `orchestrator rgw rm ...`

There is no need to artificially squeeze all services into a `orchestrator service <type>` command.

Reply:
I aggree

Reply:
That sounds like a good plan. I'll look at reworking this before we merge.

Reply:
Done